sormas-backend/src/main/java/de/symeda/sormas/backend/sample/SampleFacadeEjb.java

```diff
@@ -459,6 +459,32 @@ public void validate(SampleDto sample) throws ValidationRuntimeException {
 			throw new ValidationRuntimeException(
 				I18nProperties.getValidationError(Validations.required, I18nProperties.getPrefixCaption(SampleDto.I18N_PREFIX, SampleDto.LAB)));
 		}
+
+		if (checkAssociatedEntities) {
+			validateSampleAssociatedEntities(sample);
+		}
+
+	}
+
+	private void validateSampleAssociatedEntities(SampleDto sample) {
+		if (sample.getAssociatedCase() != null && !caseService.exists(sample.getAssociatedCase().getUuid())) {
+			throw new ValidationRuntimeException(I18nProperties.getValidationError(Validations.validCaseContactOrEventParticipant));
```

Comment:
Please add individual error messages for cases, contacts, and event participants here:

"The database does not contain a {case/contact/event participant} with this UUID."